### PR TITLE
Update configuration.markdown

### PR DESCRIPTION
**Pull request by hdboer**
## Proposed change
Further clarification of the migration from an old to a new system.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration.markdown [hunk 1]
```diff
@@ -40,6 +40,6 @@ If you can't see your integration listed there, you will need to restart Home As
 
 ## Migrating to a new system
 
-The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}.
+The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}. Once you have created the backup on the old system, you can download it to the system that is running the Home Assistant frontend. From there you can upload it to your new system using the *Upload backup* menu option of the *Backups* menu. A restore of the uploaded backup on the new system concludes the migration.
```
Comment (frenck) [inline source/_docs/configuration.markdown]: A restore of backups is offered when setting up / onboarding the new system. This path is not the "normal" path.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration.markdown (hunk 1 above)
Comment (hdboer) [reply source/_docs/configuration.markdown]: Then I would argue that a statement similar to "A restore of backups is offered when setting up / onboarding the new system" should be added to the documentation. I was preparing a migration myself and found the documentation on how to use the backup of the old system to be missing. Actually, the documentation only indicates that a backup should be made; nothing on what to do with the backup.
If you like I can add the statement to the documentation.

Comment (ArveVM) [reply source/_docs/configuration.markdown]: have a look at #26530,,, (my simplified 'Commits on Mar 31, 2023') it's adressing the same topic :)

I'm new to PR's and it seems I've c&%¤ed something up,,, 
#learningcurve :)

Comment (hdboer) [reply source/_docs/configuration.markdown]: I think I have addressed the requested changes.

**Review by frenck — APPROVED** [review]
Thanks, @hdboer 👍 

../Frenck

**Pull request merged**